chain/lf3/participation_lease.go

```diff
@@ -104,6 +104,8 @@ func (l *leaser) participate(ticket api.F3ParticipationTicket) (api.F3Participat
 	if found {
 		// short-circuite for reparticipation.
 		if currentLease == newLease {
+			newLease.ValidityTerm = newLease.ToInstance() - instant.ID
+			newLease.FromInstance = instant.ID
```

Comment:
New lease is not stored in lease map. Should it be?

Reply:
It shouldn't be stored because the lease in the lease map is the decoded ticket. It's... confusing and we should probably do something about it.